CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+Notable changes to Draft.js will be documented in this file.
+
+Changes to `src` are live in production on facebook.com at the time of release.
+
+## 0.2.0 (March 8, 2016)
+
+### Added
+
+* Move `white-space: pre-wrap` into inline style to resolve insertion issues
+* `handleDrop` prop method for `Editor` to allow manual drop management
+* `decoratedText` prop for decorator components
+* `getVisibleSelectionRect`, to provide Rect for DOM selection
+* Create Slack team inviter
+* Export `KeyBindingUtil` and `getDefaultKeyBinding`
+* Website autobuild, Travis/NPM badges
+
+### Fixed
+
+* Triple-clicks followed by block type changes now only affect first block
+* `DraftEditorLeaf` now re-renders correctly when its styles change
+* Backspace behavior within empty code blocks
+* Documentation typos, broken examples
```

Comment:
Probably don't need to call this out (same with the website / travis stuff above). That's all pretty fluid and happens out of sync with releases.